.github/workflows/test.yml

```diff
@@ -110,7 +110,12 @@ jobs:
           command: ./install.sh
 
       - name: Integration Test
-        run: pytest --cov --junitxml=junit.xml --reruns 3 _integration-test/ --customizations=${{ matrix.customizations }}
+        run: |
+          if [ "${{ matrix.compose_version }}" = "v2.0.1" ]; then
+            pytest --reruns 5 --cov --junitxml=junit.xml _integration-test/ --customizations=${{ matrix.customizations }}
```

Comment:
Let's change this back to 3, otherwise looks good.

Reply:
done